design/comments.go

```diff
@@ -100,12 +100,12 @@ var _ = a.Resource("comments", func() {
 
 	a.Action("show", func() {
 		a.Routing(
-			a.GET("/:id"),
+			a.GET("/:commentId"),
```

Comment:
@aslakknutsen I don't know but parameters in URLs typically use lower case letters, right? In the iteration resource we also use `iterationId` instead of `IterationID`.